### DISCUSSION TITLE
Shutters and presence detector alarm not finded

### DISCUSSION THREAD

**Discussion by gianpelu** [gicamm/homeassistant-comelit#22] [General]
Hello, I have installed and configured everything. I see lights, sensors, switch, scenes and alarm but I can't see shutters and presence detectors. Do you know why?
Thank you

**Comment by gicamm** [comment]
Hi, gianpelu.
About "presence detectors", do you mean the vedo IR sensor?
The shutters should work. Do you have any logs?
You should see something like 
`[custom_components.comelit.hub] added the cover`

Are you using v0.8.1 or the master?
If you used v0.8.1, you could try v0.8.2. There we made some changes to the shutters.
